sdk/core/azure-core/src/main/java/com/azure/core/implementation/util/ConfigurationUtils.java

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.implementation.util;
+
+import com.azure.core.util.ConfigurationPropertyBuilder;
+import com.azure.core.util.CoreUtils;
+
+import java.time.Duration;
+import java.util.function.Function;
+
+public class ConfigurationUtils {
+
+    private static final Function<String, Boolean> BOOLEAN_CONVERTER = (value) -> Boolean.parseBoolean(value);
+    private static final Function<String, Duration> DURATION_CONVERTER = (value) -> {
+        long timeoutMillis = Long.parseLong(value);
+        if (timeoutMillis < 0) {
+            throw new IllegalArgumentException("Duration can't be negative");
+        }
+
+        return Duration.ofMillis(timeoutMillis);
+    };
+
+    private static final Function<String, Integer> INTEGER_CONVERTER = (value) -> Integer.valueOf(value);
+    private static final Function<String, String> STRING_CONVERTER = (value) -> value;
```

Comment:
You can use the [identity function](https://docs.oracle.com/javase/8/docs/api/java/util/function/Function.html#identity--) here.
```suggestion
    private static final Function<String, String> STRING_CONVERTER = Function.identity();
```